cats/src/main/scala/org/scanamo/ScanamoCats.scala

```diff
@@ -17,19 +17,12 @@
 package org.scanamo
 
 import cats.effect.Async
-import cats.{ ~>, Monad }
+import cats.~>
 import fs2.Stream
-import org.scanamo.ops.{ CatsInterpreter, ScanamoOps, ScanamoOpsT }
+import org.scanamo.ops.CatsInterpreter
 import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
 
-class ScanamoCats[F[_]: Async](client: DynamoDbAsyncClient) {
-  final private val interpreter = new CatsInterpreter(client)
-
-  final def exec[A](op: ScanamoOps[A]): F[A] = op.foldMap(interpreter)
-
-  final def execT[M[_]: Monad, A](hoist: F ~> M)(op: ScanamoOpsT[M, A]): M[A] =
-    op.foldMap(interpreter andThen hoist)
-}
+class ScanamoCats[F[_]: Async] private (client: DynamoDbAsyncClient) extends ScanamoClient(new CatsInterpreter(client))
 
```

Comment:
Note that, in line with the other Scanamo clients, the constructor is now made `private`, and you need to use the companion object `apply()` method to create the client. So you can no longer use:

```scala
new ScanamoCats(dynamoDbAsyncClient)
```

...but instead just:

```scala
ScanamoCats(dynamoDbAsyncClient)
```